### ISSUE TITLE
catastrophic failure

### ISSUE THREAD

**Issue by itsalchemist** [c0dycode/Borderlands-Hex-Multitool#2]
i just started setting it up for bl2 but when i select borderlands 2 .exe it just pops up with the error catastrophic failure with no other information *update same with tps

**Comment by c0dycode** [comment]
Can you provide a screenshot of the errormessage?

**Comment by itsalchemist** [comment]
[Untitled document.pdf](https://github.com/c0dycode/Borderlands-Hex-Multitool/files/1756600/Untitled.document.pdf)

**Comment by c0dycode** [comment]
Ah okay, you're trying to open up the Shortcut with the Multitool. You need to guide it to the .exe-file.

It's at
```
C:\SteamLibrary\steamapps\common\Borderlands 2\Binaries\Borderlands2.exe
```

for example.

**Comment by itsalchemist** [comment]
perfect everything's working now thanks :3

**Comment by dcox0315** [comment]
hey im new to this, but im having truble with the same isue and i dont know how to do what you said to do so could you dumb it down for me

**Comment by c0dycode** [comment]
It's as simple as selecting "Open File" in any other program too.
The issue here is, that you're either selecting a shortcut to the exe (.lnk file) or the shortcut that steam creates (.url file), if you choose to create a desktop shortcut after installing a game.

Neither of these are the games exe file.
So you simply need to:
1. Click the "Select Path"-button in the Multitool.
2. Navigate to the install location of your game (see example in my comment above)
3. Double click the Borderlands2 (.exe).
4. Done

Apart from shortcuts, it'll be the only file you'll see. If you don't see any file, you either 
A) clicked the "Select Path" for the wrong game
B) You're in the wrong folder. And if you can launch your game, it exists.

The games exe will NOT be anywhere on your desktop or any of its subfolders, unless you know what you're doing.
That's as much information I can probably give you, as I do not know where you installed your game to.

**Comment by dcox0315** [comment]
that worked but ummmmmmmmmmmm now when ever i lanch it throgh steam it says epic must be opend. So do you know what to do

**Comment by c0dycode** [comment]
Sounds like you added the game (installed on epic) on steam as an external application. This obviously still requires you to launch epic beforehand.

It doesn't magically transfer the version to steam.

**Comment by dcox0315** [comment]
ok i here you and launched epic off and on steam and i still get the same mesage whitch is, epic must be running in order to launch borderlands 2. Please restart borderlands 2 after starting the Epic client.

**Comment by c0dycode** [comment]
Then you'll just have to launch the game via Epic. As I do not use the Epic version and this no longer being related to the Multitool, I can not really help with that anymore.

Good luck though!

**Comment by dcox0315** [comment]
ok thank you. do you know any one who can help with my problem

**Comment by c0dycode** [comment]
No, sorry :/
Everybody I know uses the steam version.
